include/internal/bit_utils.h

```diff
@@ -22,8 +22,11 @@
 
 CPU_FEATURES_START_CPP_NAMESPACE
 
-inline static bool IsBitSet(uint32_t reg, uint32_t bit) {
-  return (reg >> bit) & 0x1;
+#define TRUE -1
```

Comment:
Let's not go this route. Redefining TRUE is opening a can of worms.
Let's just drop this commit for now.